src_new/adapters/prebid.js

```diff
@@ -543,21 +622,26 @@ function generatedKeyCallback(adapterID, adUnits, adapterConfig, impressionID, g
 			}
 			});
 			break;
-	case "ix":
+		case "ix":
```

Comment:
![Codacy](https://codacy.pubmatic.com:443/assets/images/favicon.png) Issue found: [Expected indentation of 1 tab but found 2. (indent)](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=15196)